test/unit/bind-elements.html

```diff
@@ -468,3 +468,23 @@
     <input id="input" value$="{{inputValue}}">
   </template>
 </dom-module>
+
+
+<dom-module id="x-bind-is-attached">
+  <template>
+    <div id="check">{{isAttached}}</div>
+  </template>
+  <script>
+    Polymer({
+      is: 'x-bind-is-attached',
+      properties: {
+        isAttached: {
+          observer: '_isAttachedChanged'
+        }
+      },
+      _isAttachedChanged: function() {}
+    });
+  </script>
+</dom-module>
+
+<s
```

Comment:
Unlikely. I removed it with https://github.com/Polymer/polymer/commit/d7567b7431d07f177d89b0ee1750929df9e94686